packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styled from 'styled-components';
+import { C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_MARGIN_ABOVE_400PX,
+  GEL_SPACING_DBL,
+  GEL_SPACING,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_MINION } from '@bbc/gel-foundations/typography';
+import { getSansBold } from '@bbc/psammead-styles/font-styles';
+
+const StyledGuidanceIcon = styled.svg`
+  color: ${C_EBON};
+  fill: currentColor;
+`;
+
+const GuidanceIcon = () => (
```

Comment:
Would it make more sense to put this guidance icon in `psammead-assets`?

Reply:
@12 yeah, definitely it would. but depends on how much we'd be reusing it. was planning to build everything here first and slowly separate them out

Reply:
Any updates on what you've decided to do? I still think we should extract this out to `psammead-assets` personally...